windows/common.ps1

```diff
@@ -13,12 +13,18 @@ if ($Env:DRONE_NETRC_DEBUG) {
 if (!(Test-Path .git)) {
     Write-Host '+ git init';
     iu git init
-    
+
     Write-Host "+ git config --global --add safe.directory *"
     iu git config --global --add safe.directory '*'
-    
+
     Write-Host "+ git remote add origin $Env:DRONE_REMOTE_URL"
     iu git remote add origin $Env:DRONE_REMOTE_URL
+} else {
+    Write-Host "+ git config --global --add safe.directory *"
+    iu git config --global --add safe.directory '*'
+
+    Write-Host "+ git remote set-url origin $Env:DRONE_REMOTE_URL"
+    iu git remote set-url origin $Env:DRONE_REMOTE_URL
```

Comment:
Check if the remote URL needs to be updated before running git remote set-url.

Reply:
Done! Wrapped the git `remote set-url origin` logic into another function, which checks if origin exists. If the origin remote exists, it calls `git remote set-url origin "${origin_url}"`, otherwise it calls `git remote add origin "${origin_url}"`